example-app/src/InputDemo.js

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react'
+
+import Paper from 'ui/Paper';
+
+import Checkbox from 'ui/Input/Checkbox'
+import RadioButton from 'ui/Input/RadioButton'
+import TextField from 'ui/Input/TextField'
+import Switch from 'ui/Input/Switch'
+import SelectField from 'ui/Input/SelectField'
+import RadioButtonGroup from 'ui/Input/RadioButtonGroup'
+
+const selectOptions = [
+    { value: 1, label: 'One' },
+    { value: 2, label: 'Two' },
+    { value: 3, label: 'Three' },
+];
+
+export default class InputDemo extends Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            field1: '',
+            field2: '',
+            field3: '',
+            field4: '',
+            field5: '',
+            field6: '',
+            switch1: false,
+            switch2: false,
+            switch3: false,
+            selectValue: 0
+        }
+    }
+
+    toggleSwitch = (shard) => {
+        this.setState({
+            [shard]: !this.state[shard],
+        });
+    };
+
+    updateInputText = (shard, event) => {
+        this.setState({
+            [shard]: event.target.value,
+        });
+    };
+
+    updateSelect = selectValue => {
+        this.setState({ selectValue: parseInt(selectValue, 10) });
+    };
+
+    render() {
+        return (<Paper elevation={6} padded>
+            <RadioButtonGroup
+                options={selectOptions}
+                onChange={this.updateSelect}
+                label="I'm a label"
+                value={this.state.selectValue}
+                helpText="Choose one please"
+                error
+                inline
+            />
+            <SelectField
+                options={selectOptions}
+                label="Choose something"
+                onChange={this.updateSelect}
+                value={this.state.selectValue}
+                variant="outlined"
+                native
+                includeEmpty
+                emptyOptionText="None"
```

Comment:
```
const selectOptions = [
    { value: null, label: "None" },
    { value: 1, label: 'One' },
    { value: 2, label: 'Two' },
    { value: 3, label: 'Three' },
];
```

Reply:
You are right that two props for this are complete overkill. However, I think your last suggestion of just including it in the options is not the best way to go. 

First of all because I am making some exceptions for the empty option in the component, so it would be good to keep it separated.

But perhaps more importantly, I think keeping it separate will make for a nicer API. This ways you can use option lists that you receive (from the API) directly. Otherwise, you will continuously have to do stuff like 
```javascript
const options = [ { value: null, label: 'None' }, ...someCollectionAsArray ]
```

Reply:
Now implemented as one property: `emptyOption` (string)